apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -36,9 +36,20 @@
   <div class="bg-primary-opacity-10 py-8">
     <div class="flex flex-col px-4 gap-8 container-lg lg:mx-auto">
       <div class="flex flex-wrap justify-between sm:mb-2 ng-star-inserted">
-        <p class="text-[21px] text-title font-title" translate>
+        <p
```

Comment:
It looks like the two paragraphs (line 39 -52) are the same except of their text content. Instead of repeating, you could use one paragraph, like so:
```suggestion
        <p
  class="text-[21px] text-title font-title"
  *ngIf="selectedApiLink?.accessServiceProtocol"
  translate
>
  {{ selectedApiLink?.accessServiceProtocol === 'GPFDL' 
      ? 'record.metadata.api.form.title.gpf' 
      : 'record.metadata.api.form.title' }}
</p>
```
If we would have to change the paragraph in the future, we would only have to do that in one place :)

Reply:
Yes, you're right. I have done it.